pkgs/development/python-modules/importlib-metadata/default.nix

```diff
@@ -0,0 +1,31 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, recursivePthLoader
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  pname = "importlib_metadata";
+  version = "0.6";
+  name = "importlib_metadata";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "074v6mq5dbbr0y697fgajm503sg83faxzvwz41jxy6h0z622rc1n";
+  };
+
+  pythonPath = [ recursivePthLoader ];
```

Comment:
Why is this needed? Why don't you use `propagatedBuildInputs` or `nativeBuildInputs`?